src/actions.cpp

```diff
@@ -1899,7 +1899,7 @@ namespace
         std::string tmp = "/tmp/";
 #endif
         char sCount[12];
-        sprintf(sCount, "_%d", ++count);
+        sprintf(sCount, "_%d", ++count); /// \todo replace by std::snprintf on master
```

Comment:
Well, this is for master, so we should replace it with `snprinft` ;-)

Reply:
Done 😉